### DISCUSSION TITLE
SDK ELRS

### DISCUSSION THREAD

**Discussion by gzai** [ExpressLRS/ExpressLRS#1947] [Q&A]
is there any sdk to control quad elrs using only laptop? 
I have Jumper AION Nano TX and want to try to control only from program.

**Comment by SunjunKim** [comment]
You can emulate CRSF packets from a computer.
There's no readily available solution nor SDK, but in theory yes, you can do that.

**Reply by gzai** [reply]
thanks for the hint @SunjunKim , i'll give it a try
